colin/core/target.py

```diff
@@ -407,10 +407,11 @@ def _run_and_log(cmd, ostree_repo_path, error_msg, wd=None):
         if wd:
             kwargs["cwd"] = wd
         try:
-            subprocess.check_call(cmd, **kwargs)
+            out = subprocess.check_output(cmd, **kwargs)
```

Comment:
are you sure to use this method, thats fine, but `stderr` will still go to output instead of log

Reply:
no it won't, you didn't check the whole function :P
```
         kwargs = {
"stderr": subprocess.STDOUT,
```
https://github.com/user-cont/colin/pull/186/files#diff-4258059fcf4dc8b1c1f51b281f81b704L400

Reply:
Probably historical reasons. And what's wrong with the current approach? I agree it's not ideal, but there is nothing wrong with it, really. The `check_*` functions accept `**kwargs` anyway.

Reply:
sure :-) thats fine